src/plugins/console/server/lib/wildcard_matcher.test.js

```diff
@@ -18,7 +18,7 @@
  */
 
 /* eslint-env mocha */
```

Comment:
Just a small thing: I think we can remove those eslint-env mocha mentions

Reply:
https://github.com/elastic/kibana/pull/85514/commits/2c12c9ca5f4649ea3e5a8964bd973a3a856452aa